hcx/netextconsiderations.md

```diff
@@ -125,9 +125,9 @@ It's important to note that HCX MON will not always solve all customer issues re
 
 If the goal is to evacuate a data center on-premises, the focus should simply be on getting VMs as quick as possible to Azure VMware Solution either by Re-IP'ing VMs migrated, Duplicating Existing Networks, or simply cutting over the default gateways of networks that were stretched via L2 (See Options 1, 2, and 3).
 
-If the use case for a customer is to have a layer 2 extension in perpetuity, or for an extended period of time, then MON is a good option to enable. For this use case also customers would need to consider Network Extension High Availability (HA). This capability protects against one Network Extension appliance failure in a HA group. Network Extension HA operates without preemption, with no automatic failback of an appliance pair to the Active role. The HCX Network Extension service provides layer 2 connectivity between sites. Network Extension HA protects extended networks from a Network Extension appliance failure at either the source or remote site.
+If the use case for a customer is to have a layer 2 extension in perpetuity, or for an extended period of time, then MON is a good option to enable. For this use case also customers would need to consider Network Extension High Availability (HA). This capability protects against one Network Extension appliance failure in a HA group. Network Extension HA operates without preemption, with no automatic fallback of an appliance pair to the Active role. The HCX Network Extension service provides layer 2 connectivity between sites. Network Extension HA protects extended networks from a Network Extension appliance failure at either the source or remote site.
```

Comment:
In my understanding, this should be kept as "failback".  
https://thecontentauthority.com/blog/fallback-vs-failback

Reply:
Thanks for the resource, not familiar with the term.

Corrected